libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -1050,4 +1067,86 @@ export const buildTigerSpecificFunctions = (
+        try {
+            return await authApiCall(async (sdk) => {
+                const result = await sdk.entities.getAllEntitiesCspDirectives({});
+                return result.data?.data || [];
+            });
+        } catch (error) {
+            return [];
+        }
+    },
+    getCSPDirective: async (directiveId: string): Promise<ICSPDirective> => {
+        try {
+            return await authApiCall(async (sdk) => {
+                const result = await sdk.entities.getEntityCspDirectives({ id: directiveId });
+                return result.data?.data;
+            });
+        } catch (error: any) {
+            throw convertApiError(error);
+        }
+    },
+    createCSPDirective: async (requestData: ICSPDirective): Promise<ICSPDirective> => {
+        try {
+            const jsonApiCspDirectiveInDocument: JsonApiCspDirectiveInDocument = {
+                data: {
+                    id: requestData.id,
+                    type: JsonApiCspDirectiveInTypeEnum.CSP_DIRECTIVE,
+                    attributes: requestData.attributes,
+                },
+            };
+
+            return await authApiCall(async (sdk) => {
+                const result = await sdk.entities.createEntityCspDirectives({
+                    jsonApiCspDirectiveInDocument,
+                });
+                return result.data?.data;
+            });
+        } catch (error: any) {
+            if (error.response?.status === 400) {
+                const message = error?.response?.data?.detail
+                    ? error?.response?.data?.detail
+                    : "Server error";
+                throw new UnexpectedError(message, error);
+            }
+            throw convertApiError(error);
+        }
+    },
+    updateCSPDirective: async (directiveId: string, requestData: ICSPDirective): Promise<ICSPDirective> => {
+        try {
+            const jsonApiCspDirectiveInDocument: JsonApiCspDirectiveInDocument = {
+                data: {
+                    id: requestData.id,
+                    type: JsonApiCspDirectiveInTypeEnum.CSP_DIRECTIVE,
+                    attributes: requestData.attributes,
+                },
+            };
+
+            return await authApiCall(async (sdk) => {
+                const result = await sdk.entities.updateEntityCspDirectives({
+                    id: directiveId,
+                    jsonApiCspDirectiveInDocument,
+                });
+                return result.data?.data;
+            });
+        } catch (error: any) {
+            if (error.response?.status === 400) {
+                const message = error?.response?.data?.detail
+                    ? error?.response?.data?.detail
+                    : "Server error";
+                throw new UnexpectedError(message, error);
+            }
+            throw convertApiError(error);
+        }
+    },
+    deleteCSPDirective: async (directiveId: string) => {
```

Comment:
Please add an empty line to separate this function from the one above.

Reply:
not sure about this. Almost functions do not have an empty line to separate each one

Reply:
ok then